src/v3/test/integration/util/createAuthClient.ts

```diff
@@ -10,6 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
nit: preference for using this disable per-line. i counted 5 instances in the whole file, so it's not too bad

Reply:
used here in file scope mostly because this is only a test util function